src/main/java/org/apache/ibatis/parsing/PropertyParser.java

```diff
@@ -88,8 +88,9 @@ public String handleToken(String content) {
             return variables.getProperty(key, defaultValue);
           }
         }
-        if (variables.containsKey(key)) {
-          return variables.getProperty(key);
```

Comment:
@kazuki43zoo Just a question on this.  Historically, it was suggested to check contains before getting it for some performance reason.  Has that changed?  I have seen recently less of doing it via containsKey.

Reply:
@hazendaz Thanks for your review comment!
Your comment right!!! I was tried getting Benchmark. As result, processing using with containsKey was high performance rather than without containsKey. Therefore, I reverted my changes via c8ff596e6da0c9f8dee624994acb139f896459b2.

Reply:
学习了，看到了